docs/Changelog.md

```diff
@@ -14022,6 +14022,167 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to all numerical tensor types.</dd>
 </dl>
 
+### <a name="Gradient-12"></a>**Gradient-12**</a>
+
+  Gradient operator computes the partial derivatives of a specific tensor to
```

Comment:
Please replace "to" with "with respect to"

Reply:
Do you mind if I use `w.r.t.`? I also used in in some other places. Or it's better not to abbreviate?

Reply:
w.r.t. is fine, thank you!

Reply:
Ok. Thanks!